app/modules/import-export/module.php

```diff
@@ -489,6 +494,12 @@ private function ensure_writing_permissions() {
 		}
 	}
 
+	private function ensure_DOMDocument_exists() {
+		if ( ! class_exists( 'DOMDocument' ) ) {
+			throw new \Error( self::DOMDOCUMENT_MISSING );
+		}
+	}
+
```

Comment:
You have a mixup of two PRs. Please take these changes out of this one.
```suggestion
```

Reply:
Done.